### PR TITLE
Refactor Chigger tests

### PR DESCRIPTION
**Pull request by milljm**
Refactor a few things. Use the status bucket. Use the enumerated statuses instead of literals.

Refs #8530

### CODE REVIEW

File: python/TestHarness/testers/RunPythonApp.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import os, re
+from util import *
```
Comment (brianmoose) [inline python/TestHarness/testers/RunPythonApp.py]: `import util`

File: python/TestHarness/testers/RunException.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from util import *
```
Comment (brianmoose) [inline python/TestHarness/testers/RunException.py]: `import util`

File: python/TestHarness/testers/CheckFiles.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 from RunApp import RunApp
-from util import runCommand
+from util import *
```
Comment (brianmoose) [inline python/TestHarness/testers/CheckFiles.py]: It is usually better to do `import util` then use `util.checkOutputForPattern`

File: python/TestHarness/testers/ImageDiff.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        return params
+
+    def __init__(self, name, params):
+        RunPythonApp.__init__(self, name, params)
+
+    def prepare(self, options):
+        """
+        Cleans up image files from previous execution
+        """
+        if self.specs['delete_output_before_running'] == True:
```
Comment (brianmoose) [inline python/TestHarness/testers/ImageDiff.py]: This logic also exists in the differ testers

File: python/TestHarness/testers/RunApp.py [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import re, os, sys, time
 from Tester import Tester
+from util import *
```
Comment (brianmoose) [inline python/TestHarness/testers/RunApp.py]: `import util`

**Review by brianmoose — CHANGES_REQUESTED** [review]

File: python/TestHarness/testers/ImageDiff.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+            util.deleteFilesAndFolders(self.specs['test_dir'], self.specs['imagediff'], self.specs['delete_output_folders'])
```
Comment (brianmoose) [inline python/TestHarness/testers/ImageDiff.py]: `delete_output_folders` doesn't seem to be defined

File: python/TestHarness/testers/AnalyzeJacobian.py [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import re, os, sys
```
Comment (brianmoose) [inline python/TestHarness/testers/AnalyzeJacobian.py]: `re` isn't used

File: python/TestHarness/testers/CSVDiff.py [hunk 8]
```diff
@@ -1,5 +1,6 @@
 from RunApp import RunApp
 from CSVDiffer import CSVDiffer
+import util
 import os
```
Comment (brianmoose) [inline python/TestHarness/testers/CSVDiff.py]: `os` isn't used

File: python/TestHarness/testers/RunPythonApp.py [hunk 9]
```diff
@@ -0,0 +1,54 @@
+import os, re
```
Comment (brianmoose) [inline python/TestHarness/testers/RunPythonApp.py]: `re` isn't used

File: python/TestHarness/testers/RunPythonException.py [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+            # Process out_ok
+            if out_ok:
+                reason = ''
+
+        # Populate the bucket
+        if reason != '':
+            self.setStatus(reason, self.buck_fail)
+        else:
+            self.setStatus(self.success_message, self.buck_success)
```
Comment (brianmoose) [inline python/TestHarness/testers/RunPythonException.py]: `buck_success` doesn't seem to be defined.

File: python/TestHarness/testers/Tester.py [hunk 11]
```diff
@@ -1,4 +1,5 @@
-from util import *
+import re, os
+import util
 from InputParameters import InputParameters
```
Comment (brianmoose) [inline python/TestHarness/testers/Tester.py]: Not used

**Review by brianmoose — CHANGES_REQUESTED** [review]

**Comment by permcody** [comment]
Migrating to #8591

**Pull request closed without merging**